guides/plugins/apps/storefront/customize-templates.md

```diff
@@ -0,0 +1,99 @@
+---
+nav:
+  title: Customize templates
+  position: 10
+
+---
+
+# Customize Templates
```

Comment:
Is the uppcase in h1 heading wanted? I saw it is mixed in our docs

Reply:
H1 is always in Camel case and rest other heading followed by that is sentence case.